src/app/applicationinstancemanager.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2019  Vladimir Golovnev <glassez@yandex.ru>
+ * Copyright (C) 2013  Digia Plc and/or its subsidiary(-ies)
```

Comment:
I'm not sure, adding this line looks like a re-license from BSD to GPL2... I guess you can either use GPL2 and omit them (to claim you wrote the code), or go back to the BSD license.

Reply:
>this line looks like a re-license from BSD to GPL2... 

I just mentioned the copyright of the code I used in this file. The BSD license allows to use the code in any projects, both free and proprietary. So there are no violations.
The following license is for resulting file (applicationinstancemanager.cpp).

Reply:
>I just mentioned the copyright of the code I used in this file. 

Yes, but please read the bsd license, I'll just quote it here:
> "Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are
met:
>  * Redistributions of source code must retain the above copyright
notice, this list of conditions and the following disclaimer.
 > * Redistributions in binary form must reproduce the above copyright
notice, this list of conditions and the following disclaimer in
the documentation and/or other materials provided with the
distribution.
 > * Neither the name of Digia Plc and its Subsidiary(-ies) nor the names
of its contributors may be used to endorse or promote products derived
from this software without specific prior written permission.

If I understand correctly, the first point prohibits changing the license. And the third point means you cannot (should not?) use their name if you claim you wrote this new code.

Reply:
I'll remove that line to avoid any confusion.